docs/vsphere_ci/README.md

```diff
@@ -0,0 +1,173 @@
+# Building Windows VM images in vSphere
+
+## Goals
+This document focuses on building Windows images in vSphere 6.7 environment used in CI but can be used as example for
+other vSphere environments. We use Packer to automate installation and configuration of the Windows images. Packer 
+generates a vSphere VM template that can be converted to a VM. After the VM has been created, the machine-api can clone
+it multiple times as part of the Windows VM creation. The following steps need to be executed from the bastion instance
+or any instance that has access to the vSphere environent.
+
+## Installing Packer
+
+We assume the latest Packer has been installed. In case of our vSphere environment, we need to install Packer
+on the bastion host. The installation steps are:
```

Comment:
Please address version comment that I called out before: https://github.com/openshift/windows-machine-config-operator/pull/271#discussion_r568722379